nixos/modules/services/networking/routinator.nix

```
@@ -0,0 +1,71 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+with lib;
+let
```

Global usage of `with lib;` is an anti-pattern, because you cannot be aware of what lib imports and what it shadows.

Instead, explicitly inherit from lib what you need:

```suggestion
let
  inherit (lib)
    mkEnableOption
    mkPackageOption
    mkOption
    types
    ;
```